src/woo_publications/publications/api/serializer.py

```diff
@@ -29,14 +29,38 @@ class Meta:  # type: ignore
         )
 
 
+class EigenaarSerializer(serializers.Serializer):
+    display_name = serializers.CharField(
+        read_only=True,
+        help_text=_(
+            "obtained from the audit trail request headers, extracted from the audit trails/log entries."
+        ),
+    )
```

Comment:
```suggestion
    weergave_naam = serializers.CharField(
        source="display_name",
        read_only=True,
        help_text=_("The display name of the user, as recorded in the audit trails."),
    )
```
per the ticket specification, using Dutch attribute names in the API documentation.